bin/stashcp

```diff
@@ -85,33 +85,38 @@ function doStashCpSingle {
 	## if someone has 'Size: ' in their file path, they have bigger problems than this not working.
 	mb=$((mySz/1000000))
 	tm=$((300+mb)) ## 5 minutes + 1MBps
-	month=$(date +%m)
+	xrdcpVersion="$(xrdcp -V 2>&1)"
 
 	## First attempt
-	## Check destination directory for space
+	## Check destination directory for space in bytes
 	dirSpace=$(df -k $baseDir | tail -1 | awk '{print $3}')
-	noSpace=false
-	if [ $dirSpace -lt $mySz ]; then
-		noSpace=true
-		echo "Stashcp of $downloadFile failed - not enough space in $baseDir. $dirSpace bytes available, file is $mySz bytes."
-		exit 1
-	fi
+	dirSpace=$(echo $dirSpace *1000 | /usr/bin/bc)
```

Comment:
No reason to use bc, not floating point math.  Just use the bash's built-in `let` or `expr`